pkg/controller/plan/kubevirt.go

```diff
@@ -2081,6 +2084,9 @@ func (r *KubeVirt) libvirtDomain(vmCr *VirtualMachine, pvcs []*core.PersistentVo
 		diskSource := libvirtxml.DomainDiskSource{}
 
 		pvc := pvcsByName[vol.PersistentVolumeClaim.ClaimName]
+		if pvc == nil {
+			continue
```

Comment:
should we add a log line for this cases, in case a user/qe will need to look for this case ?